### PR TITLE
Fix broken migrations

### PR DESCRIPTION
**Pull request by mboneil10**
For the **rewrite** branch

### CODE REVIEW

**Comment by mboneil10** [comment]
The original error I was getting was cannot have a has_many :through association BusStop#routes which goes through BusStop#bus_stops_routes before through association is defined. Or something like that. Paraphrasing.

File: app/models/bus_stop.rb [hunk 1]
```diff
@@ -6,7 +6,6 @@
 class BusStop < ApplicationRecord
   has_paper_trail
 
-  has_and_belongs_to_many :routes
```
Comment (dfaulken) [inline app/models/bus_stop.rb]: But a bus stop does have and belong to many routes, though. Are we saying it's impossible to define this association now that we have a join model?

Comment (mboneil10) [reply app/models/bus_stop.rb]: Idk. We have this line https://github.com/umts/stop-project/pull/139/files#diff-a4c987345e1391c4e1f6ff9be1c6ed5aL18

Comment (dfaulken) [reply app/models/bus_stop.rb]: Oh, I see, sorry. That's fine then. We certainly can't have both.

**Pull request merged**